includes/class-wp-job-manager-shortcodes.php

```diff
@@ -158,6 +158,12 @@ public function job_dashboard_handler() {
 
 				do_action( 'job_manager_my_job_do_action', $action, $job_id );
 
+
+				$success_message = apply_filters( 'job_manager_job_dashboard_do_action', '', $action, $job_id );
```

Comment:
You have a double space above this line. I'd recommend a more specific filter name (`job_manager_job_dashboard_success_message`). Please add documentation for this new filter. 

https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#4-hooks-actions-and-filters


We should probably also consolidate `job_manager_job_dashboard_do_action_{$action}` and `job_manager_my_job_do_action`, but that can be a different PR.

Reply:
+1 on the suggested filter name. `job_manager_job_dashboard_success_message ` is much clearer IMO.